modules/redshift-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/redshift/loading/RedshiftStatements.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2012-2021 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.loader.redshift.loading
+
+// This project
+import com.snowplowanalytics.snowplow.rdbloader.common.config.ShredderConfig.Compression
+import com.snowplowanalytics.snowplow.rdbloader.discovery.{DataDiscovery, ShreddedType}
+import com.snowplowanalytics.snowplow.loader.redshift.config.RedshiftTarget
+import com.snowplowanalytics.snowplow.loader.redshift.db.Statement
+
+/**
+  * Result of discovery and SQL-statement generation steps
+  *
+  * @param dbSchema common DB schema (e.g. atomic)
+  * @param atomicCopy COPY FROM statement to load `events` table
+  * @param shredded COPY FROM statements to load shredded tables
+  * @param vacuum VACUUM statements **including `events` table** if necessary
+  * @param analyze ANALYZE statements **including `events` table** if necessary
+  * @param discovery original discovery object
+  */
+case class RedshiftStatements(
```

Comment:
It seems like this is only used by `RedshiftLoader`. Would it better to make it part of the `RedshiftLoader` instead of separate class ?